gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -924,6 +926,227 @@ public void deleteObjects(List<StorageResourceId> fullObjectNames) throws IOExce
     }
   }
 
+  /** See {@link GoogleCloudStorage#deleteFolders(List)} for details about expected behavior. */
+  @Override
+  public void deleteFolders(List<FolderInfo> folders) throws IOException {
+
+    if (folders.isEmpty()) {
```

Comment:
do you want to check for null as well or make this nonNull??

Reply:
Added @NonNull annotation